vars/prepareLogsDestination.groovy

```diff
@@ -0,0 +1,3 @@
+def call(String dest) {
+    sh "ssh ${env.LOG_SERVER_USER}@${env.LOG_SERVER} \"mkdir -p ${dest}\""

```

Comment:
hm, do we actually need to call this function explicitely, or should publishtoLogServer test if the directory exists and create it if it doesn't?